### PR TITLE
[EM-1762] add fil info entities in algolia sync

### PR DESCRIPTION
**Pull request by elvino29**

### CODE REVIEW

File: src/Normalizer/Indexer/PapCampaignNormalizer.php [hunk 1]
```diff
@@ -16,4 +17,33 @@ protected function getTitle(object $object): string
...
+    /** @param Campaign $object */
+    protected function isLocal(object $object): bool
+    {
+        return false;
+    }
+
+    /** @param Campaign $object */
+    protected function getDate(object $object): ?\DateTime
+    {
+        return $object->getCreatedAt();
```
Comment (ottaviano) [inline src/Normalizer/Indexer/PapCampaignNormalizer.php:41]: should it be `getBeginAt()` instead?

File: src/Normalizer/Indexer/PhoningCampaignNormalizer.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    /** @param Campaign $object */
+    protected function isLocal(object $object): bool
+    {
+        return false;
+    }
+
+    /** @param Campaign $object */
+    protected function getDate(object $object): ?\DateTime
+    {
+        return $object->getCreatedAt();
```
Comment (ottaviano) [inline src/Normalizer/Indexer/PhoningCampaignNormalizer.php:44]: same here, maybe beginAt?

File: src/Normalizer/Indexer/SurveyNormalizer.php [hunk 3]
```diff
@@ -16,4 +19,33 @@ protected function getTitle(object $object): string
...
+    /** @param Survey $object */
+    protected function getDescription(object $object): ?string
+    {
+        return sprintf('%s questions', $object->getQuestionsCount());
+    }
+
+    /** @param Survey $object */
+    protected function isLocal(object $object): bool
+    {
+        return SurveyTypeEnum::LOCAL === $object->getType();
```
Comment (ottaviano) [inline src/Normalizer/Indexer/SurveyNormalizer.php:37]:
```suggestion
        return $object->isLocal();
```

File: src/Normalizer/Indexer/SurveyNormalizer.php [hunk 4]
```diff
@@ -16,4 +19,33 @@ protected function getTitle(object $object): string
...
+    /** @param Survey $object */
+    protected function getDate(object $object): ?\DateTime
+    {
+        return $object->getCreatedAt();
+    }
+
+    /** @param Survey $object */
+    protected function getAuthor(object $object): ?string
+    {
+        return is_a($object, LocalSurvey::class) ? $object->getAuthor()->getFullName() : null;
```
Comment (ottaviano) [inline src/Normalizer/Indexer/SurveyNormalizer.php:49]: could you check if the author is not null?

**Review by ottaviano — APPROVED** [review]

File: src/Normalizer/Indexer/PapCampaignNormalizer.php (hunk 1 above)
Comment (elvino29) [reply src/Normalizer/Indexer/PapCampaignNormalizer.php:41]: no, I confirmed it with victor. He wants that if you change the beginAT or finishAT date, the position in the list does not change.

**Pull request merged**